pom.xml

```diff
@@ -229,6 +229,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>1.10.19</version>
+                <scope>test</scope>
```

Comment:
Double check, but I don't think we use scope in the dependencyManagement section.